etc/tls_cert_password/src/main.rs

```diff
@@ -0,0 +1,29 @@
+use std::path::PathBuf;
+
+use mongodb::{
+    bson::doc,
+    options::{Tls, TlsOptions},
+    Client,
+};
+
+#[tokio::main]
+async fn main() {
+    let mut certpath = PathBuf::from(std::env::var("DRIVERS_TOOLS").unwrap());
+    certpath.push(".evergreen/x509gen");
+    let mut options = mongodb::options::ClientOptions::parse(std::env::var("MONGODB_URI").unwrap())
+        .await
+        .unwrap();
+    options.tls = Some(Tls::Enabled(
```

Comment:
Since we're testing parsing elsewhere, I figure setting the options directly gives us slightly more coverage.